projects/js-packages/publicize-components/src/components/form/index.js

```diff
@@ -110,7 +111,9 @@ export default function PublicizeForm() {
 					<ShareCountInfo />
 					<BrokenConnectionsNotice />
 					<UnsupportedConnectionsNotice />
-					{ shouldAutoConvert && showValidationNotice && mediaId && <AutoConversionNotice /> }
+					{ shouldAutoConvert && showValidationNotice && mediaId && (
+						<AutoConversionNotice canChangeSettings={ ! isSimpleSite } />
```

Comment:
`isSimpleSite` is a function. May be you wanted to call it?
```suggestion
						<AutoConversionNotice canChangeSettings={ ! isSimpleSite() } />
```

Reply:
Fixen in f839ed86192252217553792de0398594c29319a9